scrapers/pinupdollars.yml

```diff
@@ -0,0 +1,40 @@
+name: Pinupdollars
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lanakendrick.com
+      - leannecrow.com
+      - monicamendez.com
+      - pinupfiles.com
+      - rachelaldana.com
+      - tessafowler.com
+
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class='item-title']/h1
+      Date:
+        selector: //div[@class="update-info-row"]/strong[contains(.,"Added:")]/following-sibling::text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //div[@class="update-info-block models-list-thumbs"]//a
+      Details:
+        selector: //div[@class="update-info-block"]/h3[contains(.,"Description:")]/following-sibling::text()
+        concat: "\n\n"
+      Tags:
+        Name: //ul[@class="tags"]//a
+      Image:
+        selector: //script[contains(text(),'poster')]/text()
```

Comment:
The script selector doesn't work for all sites
I tried the below and it seems to work ok
```yaml
      Image:
        selector: //script[contains(text(),'poster')]/text()|//div[@class="player-thumb"]/img/@src0_1x
        postProcess:
          - replace:
              - regex: .+(?:poster=")([^"]+).+
                with: $1
              - regex: ^
                with:
```
I havent tried yet
```yaml
        Image: //div[@class="player-thumb"]/img/@src0_1x
```
to see if it is enough for all sites